packages/fuselage/src/components/Message/MessageEmojiBase.tsx

```diff
@@ -0,0 +1,22 @@
+import type { ReactNode } from 'react';
+import React from 'react';
+
+type MessageEmojiBaseProps = {
+  name: string;
+  className?: string;
+  children?: ReactNode;
+  image?: string;
+};
+
+export const MessageEmojiBase = ({
+  name,
+  className,
+  image,
+  children,
+}: MessageEmojiBaseProps) => (
+  <div
```

Comment:
```suggestion
  <span
```

Because it could be children of `<p>`

Reply:
Yeah, I changed it's display to be inline-block so this shouldn't cause problems, but I will change because it's more semantic. Thanks for the tip!